app/src/main/java/org/oppia/app/screenshots/ScreenshotManager.kt

```diff
@@ -0,0 +1,39 @@
+package org.oppia.app.screenshots
+
+import android.Manifest.permission.WRITE_EXTERNAL_STORAGE
+import android.content.pm.PackageManager.PERMISSION_GRANTED
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat.checkSelfPermission
+import java.io.File
+import java.io.FileOutputStream
+
+/**A class that is used to take screenshots from some activity.*/
+class ScreenshotManager {
+
+  /**
+   * A function that takes a screenshot of a given activity and stores it in the
+   * application's files.
+   * @param activity The target activity to take a screenshot of.
+   */
+  fun takeScreenshot(activity: AppCompatActivity) {
```

Comment:
Don't we only need a view to be passed in?

Reply:
I have overloaded this function to receive a `View` object.  
I think we still need this one too because we can pass the `activity` directly instead of having to pass `activity.window.decorView.rootView`every time